### PR TITLE
Feature/refactoring api call

### PR DESCRIPTION
**Pull request by jhshim85**
please review Form component as I refactored handle event function.

### CODE REVIEW

File: src/Form.js [hunk 1]
```diff
@@ -4,19 +4,19 @@ import axios from 'axios';
 
 const Form = () => {
 
+    // initialize state user's input of movie title
```
Comment (beatricegeorgeslemire) [inline src/Form.js:7]: I love that you added comments!!

File: src/DisplayGifs.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 const DisplayGifs = (props)=> {
     return(
-        <div>
+        <li>
```
Comment (beatricegeorgeslemire) [inline src/DisplayGifs.js:3]: Amazing, it's more semantic. Good call!

File: src/MovieApi.js [hunk 3]
```diff
@@ -28,19 +26,13 @@ const MovieApi = (props) => {
...
-        )
-    }
-
+    return (
+        movieKeyword.slice(0,3).map((keyword)=>{
+            return (
+                <GiphyApi key={keyword.id} keyword={keyword.name} />
+            )
+        })
+    )
```
Comment (beatricegeorgeslemire) [inline src/MovieApi.js:29]: Amazing cleaning up Jay, Thank you for doing it!!

File: src/App.js [hunk 4]
```diff
@@ -5,11 +5,11 @@ import Form from './Form';
 
 function App() {
   return (
-    <div className="App">
+    <>
```
Comment (karleene) [inline src/App.js:8]: We should probably add a div with a class of wrapper, either in here or in the form component itself.

File: src/DisplayGifs.js (hunk 2 above)
Comment (karleene) [inline src/DisplayGifs.js:3]: If we have an 'li', we should also have a 'ul' above return

File: src/Form.js [hunk 5]
```diff
@@ -1,39 +1,35 @@
 import MovieApi from './MovieApi';
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
 import axios from 'axios';
 
 const Form = () => {
 
+    // initialize state user's input of movie title
     const [userInput, setUserInput] = useState('');
+    // initialize state for movie ID of searched movie with user's input
```
Comment (karleene) [inline src/Form.js:9]: Good job on the comments/pseudo code

**Review by karleene — COMMENTED** [review]
Thanks for cleaning up our code! Good job :)

**Pull request merged**